README.md

```diff
@@ -28,22 +22,29 @@ All control sequences are weighted by trajectories costs using softmax function
 
 This uses the usual ROS tools for dependency management, so please use ``rosdep`` to install the dependencies. 
 
-Note: If running on Ubuntu 20.04 or other OS's that `xtensor` is not released in in binary form, please manually install `xtensor` v 0.24.0 and `xtl` v 0.7.0. These are simply headers so the install process is trivially short, unfortunately the `xtensor` project isn't available in package managers in some common-place operating systems (albeit, all necessary ROS OS versions) so you may be required to do this yourself if building from source.
+Note: If running on Ubuntu 20.04 or other OS's that `xtensor` is not released in binary form, please manually install `xtensor` v 0.24.0 and `xtl` v 0.7.0. These are simply headers so the install process is trivially short, unfortunately the `xtensor` project isn't available in package managers in some common-place operating systems (albeit, all necessary ROS OS versions) so you may be required to do this yourself if building from source.
 
 ```
 git clone git@github.com:xtensor-stack/xtensor.git -b 0.24.0
 cd xtensor
 mkdir build
 cd build
-cmake -DCMAKE_INSTALL_PREFIX=/usr ..
-make install
+cmake ..
```

Comment:
I think adding the `-DCMAKE_INSTALL_PREFIX=/install/path` is important so that users know they should place this somewhere like `/usr` so that the cmake paths are setup to find it. Even if we remove specifically `/usr` I think its a useful command to keep in there and let them decide what the `/install/path` is for them

Reply:
That's fair.